src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -184,6 +184,29 @@ export function setupTerminalMenus(): void {
 					order: 3
 				}
 			},
+			{
+				id: MenuId.TerminalInstanceContext,
+				item: {
+					command: {
+						id: 'workbench.action.terminal.chat.start',
+						title: localize2('startChat', 'Copilot: Start in Terminal'),
+					},
+					group: '1_copilot@0',
+					order: 1
+				}
+			},
+			{
+				id: MenuId.TerminalInstanceContext,
+				item: {
+					command: {
+						id: TerminalCommandId.DisableInitialHint,
+						title: localize2('disableInitialHint', 'Disable Initial Hint'),
+					},
+					group: '1_copilot@0',
+					order: 2,
+					// when: ContextKeyExpr.and(TerminalContextKeys.initialHintEnabled),
```

Comment:
this doesn't do anything when uncommented. very strange as debugging and `dev: inspect context keys` shows it's updated correctly. 

even when I set the initial value of the context key to `true`, it's not showing up if I include a `when`. Maybe the context key is registered after this menu is created?

Reply:
nope, debugging shows the context key is created before the menu